api/src/main/java/org/openmrs/api/impl/LocationServiceImpl.java

```diff
@@ -120,7 +120,7 @@ public Location getDefaultLocation() throws APIException {
 			location = Context.getLocationService().getLocation(locationGP);
 		}
 
-		if (location == null) {
+		if (location != null) {
```

Comment:
*Perhaps* is the keyword here :/ calling `getDefaultLocation` doesn't ensure, that `location` will not be null. It's a false posivite from the Sonar.